src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split(';');
+  const result = {};
+
+  splittedString.forEach((key) => {
+    const [property, value] = key.split(':').map((element) => element.trim());
+
+    if (property && value) {
+      result[property] = value;
+    }
+  });
```

Comment:
The parameter name 'key' in the forEach callback function is misleading because it represents a single style declaration, not just a key. A more appropriate name would be 'declaration' or 'styleRule'.